### PR TITLE
gdalwarp: add page

### PR DESCRIPTION
**Pull request by JakobMiksch**
- Add the CLI tool [gdalwarp](https://gdal.org/programs/gdalwarp) from the GDAL/OGR suite.
- Version of the command: `3.2.2`

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/gdalwarp.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Reproject a raster dataset:
+
+`gdalwarp -t_srs {{EPSG:4326}} {{path/to/input.tif}} {{path/to/output.tif}}`
+
+- Crop a raster dataset by coordinates:
+
+`gdalwarp -te {{min_x}} {{min_y}} {{max_x}} {{max_y}} -te_srs {{EPSG:4326}} {{path/to/input.tif}} {{path/to/output.tif}}`
+
+- Crop a raster dataset by a vector layer:
```
Comment (navarroaxel) [inline pages/common/gdalwarp.md:14]: Perhaps this better
```suggestion
- Crop a raster dataset using a vector layer:
```

File: pages/common/gdalwarp.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# gdalwarp
+
+> Image reprojection and warping utility.
+> More information: <https://gdal.org/programs/gdalwarp>.
+
+- Reproject a raster dataset:
+
+`gdalwarp -t_srs {{EPSG:4326}} {{path/to/input.tif}} {{path/to/output.tif}}`
+
+- Crop a raster dataset by coordinates:
```
Comment (navarroaxel) [inline pages/common/gdalwarp.md:10]:
```suggestion
- Crop a raster dataset using specifics coordinates:
```

Comment (JakobMiksch) [reply pages/common/gdalwarp.md:10]:
```suggestion
- Crop a raster dataset by using specific coordinates:
```

**Review by CleanMachine1 — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
I have no idea what any of this is, but the page looks fine 😄 
Thanks!

**Pull request merged**